### PR TITLE
[Merge first] Disconnect postgres connection at fixtures

### PR DESCRIPTION
**Pull request by Warchant**
### Description of the Change

Adds teardown steps with postgres connection deactivation.

### Benefits

It fixes problems on Jenkins.

**Note**: this PR introduces fix for critical bug, should be merged ASAP.

### CODE REVIEW

File: test/module/irohad/ametsuchi/block_query_test.cpp [hunk 1]
```diff
@@ -91,6 +91,15 @@ class BlockQueryTest : public AmetsuchiTest {
     }
   }
 
+  void TearDown() override {
+    transaction->abort();
+
+    postgres_connection->deactivate();
+    postgres_connection->disconnect();
```
Comment (l4l) [inline test/module/irohad/ametsuchi/block_query_test.cpp]: Multiple disconnects?

File: test/module/irohad/ametsuchi/block_query_test.cpp [hunk 2]
```diff
@@ -304,51 +313,6 @@ TEST_F(BlockQueryTest, GetBlocksFrom1) {
  * @when read block #1
  * @then get no blocks
  */
-TEST_F(BlockQueryTest, GetBlockButItIsNotJSON) {
```
Comment (l4l) [inline test/module/irohad/ametsuchi/block_query_test.cpp]: Why these tests was removed?

Comment (Warchant) [reply test/module/irohad/ametsuchi/block_query_test.cpp]: Ops, removed a bit more that was intended

File: test/module/irohad/ametsuchi/block_query_test.cpp (hunk 1 above)
Comment (Warchant) [reply test/module/irohad/ametsuchi/block_query_test.cpp]: What? There are `deACTIVATE` and then `disCONNECT`

Comment (Warchant) [reply test/module/irohad/ametsuchi/block_query_test.cpp]: What? There are `deACTIVATE` and then `disCONNECT`

Comment (l4l) [reply test/module/irohad/ametsuchi/block_query_test.cpp]: I mean this pair and `TearDown` call from `AmetsuchiTest` below

Comment (Warchant) [reply test/module/irohad/ametsuchi/block_query_test.cpp]: These are two different connections.

File: test/module/irohad/ametsuchi/block_query_test.cpp [hunk 3]
```diff
@@ -91,6 +91,15 @@ class BlockQueryTest : public AmetsuchiTest {
     }
   }
 
+  void TearDown() override {
+    transaction->abort();
```
Comment (lebdron) [inline test/module/irohad/ametsuchi/block_query_test.cpp]: https://github.com/jtv/libpqxx/blob/5b17abce5ac2b1a2f8278718405b7ade8bb30ae9/include/pqxx/nontransaction.hxx#L56
```
 * Any query executed in a nontransaction is committed immediately, and neither
 * commit() nor abort() has any effect.
```

File: test/module/irohad/ametsuchi/block_query_test.cpp [hunk 4]
```diff
@@ -91,6 +91,15 @@ class BlockQueryTest : public AmetsuchiTest {
...
+
+    postgres_connection->deactivate();
```
Comment (lebdron) [inline test/module/irohad/ametsuchi/block_query_test.cpp]: `pqxx::connection_base::disconnect()` contains `m_Conn = m_policy.do_disconnect(m_Conn);`
`pqxx::connection_base::deactivate()` also contains `  m_Conn = m_policy.do_disconnect(m_Conn);`
So does `pqxx::connection_base::close()`, and it is called in destructor, so manual calls are not necessary.

**Pull request closed without merging**